caikit/core/data_model/streams/data_stream.py

```diff
@@ -337,22 +338,20 @@ def from_header_csv(cls, filename: str, *args, **kwargs) -> "DataStream[Dict]":
             error(
                 "<COR44308234E>",
                 FileNotFoundError(
-                    "csv filename `{}` does not exist or is not a regular file.".format(
-                        filename
-                    )
+                    "csv filename %s does not exist or is not a regular file.", filename
```

Comment:
Shoot, this actually _doesn't_ work. When instantiating an exception, using `.format` (or the old school `"foo %s" % "bar"` syntax) to explicitly do the formatting is what you want to do. It's only in arguments to logging functions that you want to use `,` to delay the interpolation